src/main/java/games/strategy/engine/config/PropertyFileReader.java

```diff
@@ -27,14 +27,22 @@
 
 
   String readProperty(final String propertyKey) {
+    return readProperty(propertyKey, true);
+  }
+
+  String readProperty(final String propertyKey, boolean throwException) {
```

Comment:
@DanVanAtta Your PR is likely going to be merged first, I will update my code to use the "new system" accordingly